dbt/include/bigquery/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,18 @@
+{% macro bigquery__get_columns_spec_ddl() %}
+  {# loop through user_provided_columns to create DDL with data types and constraints #}
+  {% if config.get('constraints_enabled', False) %}
+    {%- set user_provided_columns = model['columns'] -%}
+    (
+    {% for i in user_provided_columns -%}
+      {%- set col = user_provided_columns[i] -%}
+      {% set constraints = col['constraint'] -%}
+      {%- set checks = col['check'] -%}
+      {%- if checks -%}
+        {{exceptions.warn("We noticed you have `check` in your configs, these are NOT compatible with BigQuery and will be ignored")}}
+      {%- endif %}
+      {{ col['name'] }} {{ col['data_type'] }} {{ col['constraint']  or "" }} {{ "," if not loop.last }}
```

Comment:
Update this code using this example as a reference: https://github.com/dbt-labs/dbt-snowflake/pull/341/files#diff-c96dce207e585a460348ff3f7fbd4c39d5822de2b13e2443648c0d8948066178R13

Reply:
done, I had the wrong config therefore that code was traversing the word instead of the list. Works now!